### ISSUE TITLE
mockServer.get not working

### ISSUE THREAD

**Issue by burritoIand** [embermap/ember-cli-fastboot-testing#313]
Hello,

I'm trying to set up a fastboot rendering test and running into a problem where I can't seem to get mockServer.get() to work, and the test is instead hitting my real api.

my real api looks like: https://...api.com/v1/posts/:id

in the test I tried:
`await mockServer.get('https://...api.com/v1/posts/abc123', {`
`await mockServer.get('/v1/posts/abc123', {`
`await mockServer.get('v1/posts/abc123', {`
`await mockServer.get('/posts/abc123', {`
`await mockServer.get('posts/abc123', {`

How might I be able to further debug this? I turned off mirage autostart so mirage shouldnt be running either..

test command:
`ember t -s -m 'FastBoot | routes/post'`

Separate question: Is it expected to have to add `0.0.0.0:7357` to the fastboot whitelist in config? I had to do that to get past a separate error message.

**Comment by ryanto** [comment]
Yikes, sorry about that. Do you know if your Ember application is making requests to localhost, or another origin? Any special host configuration in your Ember Data adapter?

**Comment by burritoIand** [comment]
np! Yes I have host/namespace set in my application adapter and requests are made to that:
```
  @reads('fetch.headers') headers;

  coalesceFindRequests = true;
  host = config.api;
  namespace = config.namespace;
```
where host = 'https://myapi.com'
and namespace = 'v1/'

**Comment by ryanto** [comment]
You might need the fixes in this PR https://github.com/embermap/ember-cli-fastboot-testing/pull/311

I'll get this merged and released shortly. For now trying pointing to that branch to see if it works.

**Comment by burritoIand** [comment]
ok great will try, thank you!

**Comment by ryanto** [comment]
Awesome. That PR has been merged and released as `0.3.0`.

I'm going to close this issue for now, but feel free to reopen if it doesn't solve your problem.

**Comment by burritoIand** [comment]
![image](https://user-images.githubusercontent.com/230757/80686897-4f598280-8a7e-11ea-9f30-f88fed0e286c.png)

in business! thank you sir

tangential: could this ever be somehow merged with mirage config router?
